examples/example_llama3.py

```diff
@@ -83,7 +89,27 @@ def input_fn():
     return x
 
 
-configure_inductor_for_autobucketing("aten")
+autobucketing_level = "aten"
+
+if autobucketing_level == "aten":
+    aten_autobucketing_config.custom_runtime_estimation = (
+        make_custom_runtime_estimation(mesh)
+    )
+    # this is from the stacked pr in https://github.com/pytorch/pytorch/pull/163960
+    torch._inductor.config.reorder_for_peak_memory = False
+    torch._inductor.config.reorder_for_compute_comm_overlap = False
+    aten_autobucketing_reordering_pass = partial(
+        aten_autobucketing_reordering_pass,
+        configs=aten_autobucketing_config,
+    )
+    torch._inductor.config.post_grad_custom_post_pass = (
+        aten_autobucketing_reordering_pass
+    )
```

Comment:
heads up @wconstab as I've moved back this part of the code in the example.

The reason is that there are some flags which are not exposed in the PyTorch configs, so we need to wrap those manually.

cc @eellison